src/front/shared/redux/actions/ethLikeAction.ts

```diff
@@ -615,6 +615,17 @@ export default {
     adminFeeObj: externalConfig.opts?.fee?.bnb,
     web3: new Web3(providers.binance_provider),
   }),
+  CNDL: new EthLikeAction({
+    coinName: 'Candle',
+    ticker: 'CNDL',
+    privateKeyName: 'eth',
+    chainId: externalConfig.evmNetworks.CNDL.chainId,
+    explorerApiName: 'candlescan',
```

Comment:
`candlescan` -> `cndlscan`, because you use this key in 2 places: mainnet `link.js` and tesnet `api.js`